### PR TITLE
v0.0.2

### PR DESCRIPTION
**Pull request by danilkolikov**
New version of the smart contract:

- Links can be stored in the blockchain
- Links can be retrieved from it

### CODE REVIEW

File: contracts/LinkStorage.sol [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  function getLinkAt(uint index) external view returns (
+    bool exists,
+    string url,
+    string label,
+    string description,
+    string hashtags,
+    uint timestamp
+  ) {
+    exists = false;
+    if (index < links.length) {
```
Comment (zerogerc) [inline contracts/LinkStorage.sol:38]: what if index > links.length ?

File: contracts/Utils.sol [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+contract Owned {
+  address owner;
+
+  function Owned() public {
+    owner = msg.sender;
+  }
+
+  modifier onlyowner() {
+    if (msg.sender == owner) {
+      _;
```
Comment (zerogerc) [inline contracts/Utils.sol:12]: What does this means?)

**Review by zerogerc — APPROVED** [review]

File: contracts/LinkStorage.sol (hunk 1 above)
Comment (danilkolikov) [reply contracts/LinkStorage.sol:38]: `exists` will be equal to false, and I handle it on API side

File: contracts/Utils.sol (hunk 2 above)
Comment (danilkolikov) [reply contracts/Utils.sol:12]: modifier can be applied to a function, and it's body will be placed where `_` is located

**Pull request merged**